### PR TITLE
pdctl: add store label and restore store command documents 

### PR DESCRIPTION
**Pull request by nolouch**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
https://github.com/tikv/pd/pull/5510

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -1276,47 +1278,70 @@ store 1
...
-store label 1 zone cn
+store label 1 zone=cn
+```
+
+更新 store id 为 1 的 label，并添加一个新的 label：
+
+{{< copyable "" >}}
+
+```bash
+store label 1 zone=us disk=hdd
```
Comment (rleungx) [inline pd-control.md:1319]: I think it looks the same as the rewrite.

**Review by rleungx — APPROVED** [review]

File: pd-control.md [hunk 2]
```diff
@@ -1327,8 +1352,11 @@ store weight 1 5 10
 
 {{< copyable "" >}}
 
+#### store limit 命令
+
+详细解释：[store limit](/configure-store-limit.md)
```
Comment (Oreoxmt) [inline pd-control.md:1357]:
```suggestion
关于 `store limit` 的原理和使用方法，请参考 [`store limit`](/configure-store-limit.md)。
```

File: pd-control.md [hunk 3]
```diff
@@ -1276,47 +1278,72 @@ store 1
...
 
 {{< copyable "" >}}
 
 ```bash
-store delete 1
+store delete 1 // 下线一台 store 
 ```
 
-撤销已使用 store delete 下线并处于 Offline 状态的 store。撤销后，该 store 会从 Offline 状态变为 Up 状态。注意，该命令无法使 Tombstone 状态的 store 变回 Up 状态。以下示例撤销已使用 store delete 下线的 store，其 store id 为 1：
+撤销已使用 `store delete` 下线并处于 Offline 状态的 store。撤销后，该 store 会从 Offline 状态变为 Up 状态。注意，该命令无法使 Tombstone 状态的 store 变回 Up 状态。以下示例撤销已使用 `store delete` 下线的 store id 为 1 的 store，并删除所有 Tombstone 状态的 store：
```
Comment (ran-huang) [inline pd-control.md:1291]:
```suggestion
你可以撤销已使用 `store delete` 下线并处于 Offline 状态的 store。撤销后，该 store 会从 Offline 状态变为 Up 状态。注意，该命令无法使 Tombstone 状态的 store 变回 Up 状态。以下示例撤销已使用 `store delete` 下线的 store id 为 1 的 store，并删除所有 Tombstone 状态的 store：
```

File: pd-control.md [hunk 4]
```diff
@@ -1276,47 +1278,72 @@ store 1
...
+>> store cancel-delete 1                // 取消下线
+>> store remove-tombstone              // 删除所有 tombstone 状态的 store
 ```
 
 > **注意：**
 >
 > 若下线过程中切换了 PD leader，需要手动修改 store limit。
 
-设置 store id 为 1 的 store 的键为 "zone" 的 label 的值为 "cn"：
+#### store label 命令
```
Comment (ran-huang) [inline pd-control.md:1304]:
```suggestion
#### 管理 store label
```

File: pd-control.md [hunk 5]
```diff
@@ -1276,47 +1278,72 @@ store 1
...
+
+`store label` 用于管理 store label。
```
Comment (ran-huang) [inline pd-control.md:1306]:
```suggestion
`store label` 命令用于管理 store label。
```

File: pd-control.md [hunk 6]
```diff
@@ -1327,8 +1354,11 @@ store weight 1 5 10
 
 {{< copyable "" >}}
```
Comment (Oreoxmt) [inline pd-control.md:1356]:
```suggestion

```

File: pd-control.md [hunk 7]
```diff
@@ -1325,10 +1352,11 @@ store label 1 --force
 store weight 1 5 10
 ```
 
-{{< copyable "" >}}
+#### store limit 命令
```
Comment (Oreoxmt) [inline pd-control.md:1355]:
```suggestion
#### `store limit` 命令
```

File: pd-control.md [hunk 8]
```diff
@@ -1284,39 +1288,62 @@ store 1
...
+```bash
+store label 1 disk --delete
 ```
 
 > **注意：**
 >
 > - store 的 label 更新方法使用的是合并策略。如果修改了 TiKV 配置文件中的 store label，进程重启之后，PD 会将自身存储的 store label 与其进行合并更新，并持久化合并后的结果。
 > - 如果希望使用 TiUP 统一管理 store label 的话，可以在集群重启前，使用 PD Control 的 `store label <id> --force` 命令将 PD 存储的 store label 清空。
 
+#### store weight 命令
```
Comment (Oreoxmt) [inline pd-control.md:1345]:
```suggestion
#### `store weight` 命令
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: pd-control.md [hunk 9]
```diff
@@ -1284,39 +1288,62 @@ store 1
...
+通过 `--rewrite` 重写 store id 为 1 的所有 label，之前有的 label 会被覆盖：
 
 {{< copyable "" >}}
 
 ```bash
-store label 1 --force
+store label 1 region=us-est-1 disk=ssd --rewrite
+```
+
+删除指定 store 的对应 label：
```
Comment (shichun-0415) [inline pd-control.md:1332]:
```suggestion
删除 id 为 1 的 store 的指定 label：
```

File: pd-control.md [hunk 10]
```diff
@@ -1284,39 +1288,62 @@ store 1
...
+```bash
+store label 1 disk --delete
 ```
 
 > **注意：**
 >
 > - store 的 label 更新方法使用的是合并策略。如果修改了 TiKV 配置文件中的 store label，进程重启之后，PD 会将自身存储的 store label 与其进行合并更新，并持久化合并后的结果。
 > - 如果希望使用 TiUP 统一管理 store label 的话，可以在集群重启前，使用 PD Control 的 `store label <id> --force` 命令将 PD 存储的 store label 清空。
 
+#### `store weight` 命令
```
Comment (shichun-0415) [inline pd-control.md:1345]:
```suggestion
#### 设置 weight
```

File: pd-control.md [hunk 11]
```diff
@@ -1284,39 +1288,62 @@ store 1
...
+
 设置 store id 为 1 的 store 的 leader weight 为 5，Region weight 为 10：
```
Comment (shichun-0415) [inline pd-control.md:1347]:
```suggestion
将 id 为 1 的 store 的 leader weight 设为 5，Region weight 设为 10：
```

File: pd-control.md [hunk 12]
```diff
@@ -1325,10 +1352,11 @@ store label 1 --force
 store weight 1 5 10
 ```
 
-{{< copyable "" >}}
+#### `store limit` 命令
+
+关于 `store limit` 的原理和使用方法，请参考 [`store limit`](/configure-store-limit.md)。
```
Comment (shichun-0415) [inline pd-control.md:1357]:
```suggestion
#### 设置调度限速

关于 `store limit` 的原理和使用方法，请参考 [`store limit`](/configure-store-limit.md)。
```

File: pd-control.md [hunk 13]
```diff
@@ -1276,59 +1274,78 @@ store 1
...
+将 id 为 1 的 store 的 leader weight 设为 5，Region weight 设为 10：
 
 ```bash
 store weight 1 5 10
 ```
 
-{{< copyable "" >}}
+#### 设置 store 调度限速
+
+关于 `store limit` 的原理和使用方法，请参考 [`store limit`](/configure-store-limit.md)。
```
Comment (shichun-0415) [inline pd-control.md:1346]:
```suggestion
通过 `store-limit`，你可以设置 store 的调度速度。关于 `store limit` 的原理和使用方法，请参考 [`store limit`](/configure-store-limit.md)。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Pull request merged**